src/mmw/js/src/compare/views.js

```diff
@@ -41,6 +42,18 @@ var CompareWindow = Marionette.LayoutView.extend({
         // so the offset of the container needs to be
         // recomputed.
         $(window).bind('resize.app', _.debounce(_.bind(this.updateContainerPos, this)));
+        this.listenTo(App.user, 'change:guest', this.saveAfterLogin);
+    },
+
+    saveAfterLogin: function(user, guest) {
+        if (!guest && this.model.isNew()) {
+            var user_id = user.get('id');
+            this.model.set('user_id', user_id);
+            this.model.get('scenarios').each(function(scenario) {
+                scenario.set('user_id', user_id);
```

Comment:
Should we move this into `saveProjectAndScenarios` ? Seems like we'd always want this to happen based off of the project.